packages/dev/biome-config/configs/biome-base.json

```diff
@@ -0,0 +1,147 @@
+{
+	"$schema": "../node_modules/@biomejs/biome/configuration_schema.json",
+	"organizeImports": {
+		"enabled": true
+	},
+	"json": {
+		"formatter": {
+			"enabled": true,
+			"trailingCommas": "none",
+			"lineWidth": 100,
+			"indentStyle": "space",
+			"indentWidth": 2,
+			"lineEnding": "lf"
+		}
+	},
+	"javascript": {
+		"globals": ["vitest"],
+		"formatter": {
+			"enabled": true,
+			"arrowParentheses": "always",
+			"bracketSpacing": true,
+			"indentWidth": 2,
+			"indentStyle": "space",
```

Comment:
🟡 This will be hell to migrate. We're using tab in all of our FE projects. Should we stick to [default tab configuration](https://biomejs.dev/reference/configuration/#formatterindentstyle)?

Reply:
sure

Reply:
Sure we can. Is BE using spaces by default (at Lokalise)?

Reply:
yup, with the exception of Autopilot

Reply:
Should we also create an override for AP(backend)? or do we want to migrate to space?

Reply:
last time I ran a poll on this topic, 100% of AP engineers who participated selected "I don't care" option, so it doesn't look like we need to maintain two separate sets of rules, we can just reformat it once and apply it within open PRs as well

Reply:
Sounds great to me! 🙏 thanks for sharing Igor 🙇